ViAn/Video/shapes/shape.h

```diff
@@ -20,7 +20,8 @@ class Shape {
     static cv::Scalar qcolor_to_scalar(QColor col);
     static cv::Point qpoint_to_point(QPoint pnt);
 
-    static const int LINE_THICKNESS = 2;
+    static const int LINE_THICKNESS = 2; // Constant used for the thickness of the drawn shapes.
+    static constexpr double ALPHA = 0.6; // Costant used for the opacity.
 
 protected:
     cv::Scalar colour;
```

Comment:
I also had to remove a reset on the include/exclude to make it work with the checkbox. And I fixed a bug where you could not clear or undo the analysis area while the video was playing.